Made the first version for html form css

checkmate-bitch opened the pull request.

tamj0rd2 committed:
manifest.json: Include a default popup

The popup is what displays when you click on the extension's icon.

Improves https://github.com/tamj0rd2/reddit_search/issues/4

tamj0rd2 committed:
popup.html: Add basic html to popup

Add the standard html boilerplate to the popup.

Improves https://github.com/tamj0rd2/reddit_search/issues/4

tamj0rd2 committed:
popup.html: Add a form

Include input for google domain name, subreddit and
a search query.

Fixes https://github.com/tamj0rd2/reddit_search/issues/4

checkmate-bitch committed:
Added the styling for popup.html

checkmate-bitch committed:
Added the styling for popup.html

checkmate-bitch committed:
Modified files are staged

checkmate-bitch committed:
Initial commit

checkmate-bitch committed:
Initial commit

checkmate-bitch commented:
I had to add these to the master branch as the popup branch is causing some issues.

checkmate-bitch committed:
randomtext.txt removed

checkmate-bitch committed:
randomtext.txt removed

tamj0rd2 commented:
@checkmate-bitch Your repo isn't up to date so I won't be able to merge this yet.

You can use the methods here to keep your fork up to date: http://stackoverflow.com/a/28505516

tamj0rd2 committed:
popup.html: Include bootstrap

So that we can add better styling.

Improves https://github.com/tamj0rd2/reddit_search/issues/6

tamj0rd2 committed:
popup.html: Move form to container

Improves https://github.com/tamj0rd2/reddit_search/issues/6

tamj0rd2 committed:
popup.html: Add html lang

tamj0rd2 committed:
popup.html: Remove name from inputs

The name attr isn't required.

tamj0rd2 committed:
popup.html: Add classes to form

Improves https://github.com/tamj0rd2/reddit_search/issues/6

tamj0rd2 committed:
popup: Set container's min-width

The input boxes were way too small due to the container
only being the width of the labels.

Improves https://github.com/tamj0rd2/reddit_search/issues/6

checkmate-bitch committed:
Resolved the conflict due to rebasing

checkmate-bitch committed:
popup.html: Used tamj0rd's file

checkmate-bitch committed:
popup.html: resolved the conflicts by using tamj0rd's version of file

tamj0rd2 commented:
Project is now using bootstrap, so custom styling doesn't need to be used anymore.
